fusor-ember-cli/app/controllers/review/progress/overview.js

```diff
@@ -52,6 +53,44 @@ export default Ember.Controller.extend(ProgressBarMixin, NeedsDeploymentMixin, {
     }
 
     return true;
-  })
+  }),
+
+  isAbandonModalOpen: false,
+  loadingRedeployment: false,
+
+  actions: {
+    redeploy() {
+      this.set('loadingRedeployment', true);
 
+      let depl = this.get('deploymentController.model');
+      let token = Ember.$('meta[name="csrf-token"]').attr('content');
+
+      request({
+        url: '/fusor/api/v21/deployments/' + depl.get('id') + '/redeploy',
+        type: "PUT",
+        headers: {
+          "Accept": "application/json",
+          "Content-Type": "application/json",
+          "X-CSRF-Token": token,
+          "Authorization": "Basic " + this.get('session.basicAuthToken')
+        }
+      }).then((response) => {
+        let newTaskUUID = response.id;
+        depl.set('foreman_task_uuid', newTaskUUID);
+        depl.set('has_content_error', false);
+        depl.save();
+        this.send('refreshModelOnOverviewRoute');
+      }).catch((err) => {
+        console.log('ERROR occurred attempting a redeploy', err);
+      }).finally(() => this.set('loadingRedeployment', false));
+    },
+    abandon() {
+      this.set('isAbandonModalOpen', true);
+    },
+    executeAbandonment() {
+      let depl = this.get('deploymentController.model');
+      depl.destroyRecord();
```

Comment:
@eriknelson, It's not obvious to me that "abandon" means "Cancel and Delete".  I was expecting just "Cancel". Maybe have 3 buttons, just like the modal `cancel-deployment-modal`. Or better yet, maybe use that modal by passing in a variables for the title and body text. Then, we don't need another modal component.